samtranslator/model/sam_resources.py

```diff
@@ -2081,8 +2088,9 @@ def _construct_sqs_queue_policy(
         queue_policy_name = (
             f"{self.logical_id}QueuePolicyDestination{dest_index}" if multi_dest else f"{self.logical_id}QueuePolicy"
         )
+        # For multi-destination connector, modifying resource attribute will affect others, use deep copy to prevent it
```

Comment:
What do you mean here?

Reply:
removed. refactored to use another method